App/views.py

```diff
@@ -0,0 +1,55 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+import json
+import requests
+import toolforge
+
+def main2(request):
+    if(request.method=='POST'):
+        usr = request.POST['username']
+        SqlQuery1 = "SELECT user_editcount FROM user WHERE user_name='" + usr + "';"
+        SqlQuery2 = "SELECT ss_total_edits FROM site_stats"
+        SqlQuery3 = "select rank from (select user_name, @curRank := @curRank + 1 as rank from user p, ( select @curRank := 0 ) q order by user_editcount DESC) q where user_name='"+usr+"';"
+        SqlQuery4 = "select count(*) from user;"
+        conn = toolforge.connect('kawiki_p')
+        with conn.cursor() as curr:
+            s1 = curr.execute(SqlQuery1)
+            useredits_tup = curr.fetchall()
+            s2 = curr.execute(SqlQuery2)
+            totaledits_tup =curr.fetchall()
+            s3 = curr.execute(SqlQuery3)
+            userrank_tup = curr.fetchall()
+            s4 = curr.execute(SqlQuery4)
+            totalusers_tup = curr.fetchall()
+
+        conn.close()
+        useredits = useredits_tup[0][0]
+        totaledits = totaledits_tup[0][0]
+        userrank = userrank_tup[0][0]
+        totalusers = totalusers_tup[0][0]
+
+        percentile = 100-((userrank*100)/(totalusers+1))
+        rating = (userrank/totalusers)*100
+        if(rating<=1):
```

Comment:
This big if...else structure could be written much more simply.

Reply:
Replaced.